packages/stack/src/TransitionConfigs/TransitionPresets.tsx

```diff
@@ -150,3 +156,17 @@ export const ModalTransition = Platform.select({
   ios: ModalPresentationIOS,
   default: BottomSheetAndroid,
 });
+
+type AnimationTransitionMap = {
+  [key in StackAnimationTypes]: TransitionPreset;
+};
+
+export const AnimationTransitions: AnimationTransitionMap = {
+  default: DefaultTransition,
+  fade: ModalFadeTransition,
+  fade_from_bottom: FadeFromBottomAndroid,
+  none: DefaultTransition,
+  slide_from_bottom: BottomSheetAndroid,
+  slide_from_left: SlideFromLeftIOS,
+  slide_from_right: SlideFromRightIOS,
+};
```

Comment:
Instead of an annotation, it's simpler to let TS infer the type:

```suggestion
export const AnimationTransitions = {
  default: DefaultTransition,
  fade: ModalFadeTransition,
  fade_from_bottom: FadeFromBottomAndroid,
  none: DefaultTransition,
  slide_from_bottom: BottomSheetAndroid,
  slide_from_left: SlideFromLeftIOS,
  slide_from_right: SlideFromRightIOS,
} as const;
```